.genx/configure.js

```diff
@@ -91,18 +104,24 @@ module.exports = async (data, utils) => {
   /**
    * Configuring server projects
    */
-   writeFileWithData(path.resolve(directory, "settings.gradle.kts"), data);
-   writeFileWithData(path.resolve(directory, "docker-compose.yml"), data);
+   writeFile(path.resolve(directory, "settings.gradle.kts"), data);
+   writeFile(path.resolve(directory, "docker-compose.yml"), data);
 
   const serverJvmRoot = path.resolve(directory, "server", "jvm");
   const appNamePlaceholder = "genesis";
   const dictionaryCacheRoot = path.resolve(directory, "server", "jvm", "genesis-dictionary-cache");
 
-  utils.listAllFiles(serverJvmRoot, toFilter => {
-    return !toFilter.path.endsWith(".jar")
-  }).forEach(filteredPath => {
-    writeFileWithData(filteredPath, data);
-  });
+  const listAllFilesIncSubDirs = (rootFolder, fileFilter, incSubDirectories) => {
+    return klawSync(rootFolder, {
+        nodir: false,
+        filter: fileFilter,
+        traverseAll: incSubDirectories
+    }).map((i) => i.path);
```

Comment:
I guess you might've needed `traverseAll` here because filter below is `item.path.endsWith(".kts")`. If `item` has a flag to indicate it's a dir, filter could've been `item ends with kts || is directory` and just `nodir` would've been sufficient. In any case, for the actual shared function I'd just expose an options object as the last param, so people could set whatever permutations of `nodir` and `traverseAll` flags they needed.

Reply:
Well, that was an overzealous filter anyway, my dockerfile is broken because it's not part of the filter.
Think it just needs the check for `.DS_Store` and `.gradle` dirs, so I'll try removing traverseAll again